src/main/java/fr/uge/chargepointconfiguration/user/NewUserDto.java

```diff
@@ -0,0 +1,19 @@
+package fr.uge.chargepointconfiguration.user;
+
+/**
+ * DTO to create an user in the database.
+ *
+ * @param firstName first name of the user.
+ * @param lastName last name of the user.
+ * @param email email of the user.
+ * @param password password encrypted of the user.
+ * @param role role of the user.
+ */
+public record NewUserDto(
```

Comment:
```suggestion
public record CreateUserDto(
```
rename for consistency with other create DTO